src/Security/TwoFactor/Repository/TrustedBrowser.php

```diff
@@ -11,6 +11,9 @@
 use Friendica\Database\Database;
 use Friendica\Security\TwoFactor\Exception\TrustedBrowserNotFoundException;
 use Friendica\Security\TwoFactor\Exception\TrustedBrowserPersistenceException;
+use Friendica\Security\TwoFactor\Collection\TrustedBrowsers as TrustedBrowsersCollection;
+use Friendica\Security\TwoFactor\Factory\TrustedBrowser as TrustedBrowserFactory;
+use Friendica\Security\TwoFactor\Model\TrustedBrowser as TrustedBrowserModel;
```

Comment:
Do we still need the `use Friendica\Security\TwoFactor;` statement on line 10?

Reply:
No, it is removed now.